src/amber/router/route_set.cr

```diff
@@ -176,10 +176,11 @@ module Amber::Router
       segments = split_path path
       matches = select_routes(segments)
 
-      if matches.size > 1
-        matches.sort.first
-      elsif matches.size == 0
+      case matches.size
+      when 0
         RoutedResult(T).new nil
+      when .> 1
```

Comment:
Omg, i tried to do something like this before and never thought to add the `.` :neutral_face:

Reply:
I reckon we all do have our TIL moments ;)